### ISSUE TITLE
Python: max_tokens not read from config.json in Python

### ISSUE THREAD

**Issue by fabiofumarola** [microsoft/semantic-kernel#4841]
**Describe the bug**
I have found that providing a skill with a skprompt.txt and a config.json the parameter max_tokens is not used during the generation

**To Reproduce**
1. create a folder test_skill
2. add the following config
```
{
    "schema": 1,
    "type": "completion",
    "description": "generate a story",
    "completion": {
        "max_tokens": 10,
        "temperature": 0.5
    },
    "input": {
        "parameters": [
            {
                "name": "input",
                "description": "generate a story",
                "defaultValue": ""
            }
        ]
    }
}
```
4. add the following prompt
```generate a story about unicorns.```

**Expected behavior**
the generated text should be "Once upon a time, in a magical land far", 
while the text is "Once upon a time, in a magical land far away, there lived a herd of unicorns. These unicorns were unlike any other creatures in the land, for they possessed the power of healing. Their horns glowed with a soft, golden light that could cure any ailment or injury.

The unicorns lived in a lush forest filled with sparkling streams and towering trees. They spent their days grazing on sweet grasses and playing in the sunshine. But one day, a dark cloud descended upon the land. A wicked sorcerer had come to the forest, seeking to capture the unicorns and steal their healing powers.

The unicorns knew they had to act fast to protect themselves and their magic. They gathered together and devised a plan to outsmart the sorcerer. They would use their powers of illusion to create a false herd, leading the sorcerer away from their true home.

The unicorns worked together, using their magic to create a shimmering mirage of themselves. The sorcerer, seeing the false herd, chased after them, believing he had found the real unicorns. But as he drew closer, the mirage vanished, leaving him lost and confused in the forest.

Meanwhile, the true unicorns had fled to a secret glade deep in the heart of
"

**Screenshots**

**Platform**
 - OS: Mac
 - IDE: VS Code
 - Language: Python
 - Source: pip version0.4.7.dev0

**Additional context**
the problem is in the load of the

**Comment by moonbox3** [comment]
Hi @fabiofumarola, as of SK Python v0.4.7, we've moved away from supporting `skills`. You should now please use `plugins`. You will notice the structure of the [config.json](https://github.com/microsoft/semantic-kernel/blob/cd852e0f49958252ce33a39fe58794d8f5dee786/samples/plugins/WriterPlugin/StoryGen/config.json#L4C4-L4C22) is different. It has `execution_settings` with a key of `default` and then the config values.

You can use the kernel's `import_semantic_plugin_from_directory` and it will create a `KernelPlugin` from the `config.json` and `skprompt.txt`.

**Comment by moonbox3** [comment]
Closing this issue. Please re-open if you need help with this specific item. Otherwise, please feel free to create a new issue and we will address it.
